robottelo/common/helpers.py

```diff
@@ -229,7 +229,7 @@ def invalid_values_list():
     code.
 
     """
-    return ['', ' ', '   '] + invalid_names_list()
+    return ['', ' ', '\t'] + invalid_names_list()
```

Comment:
Supposed to be a tab symbol here. Probably IDE autoreplaced it with 3 spaces. Anyway, `\t` should work for sure.